lib/mvn-wrapper-legacy.ts

```diff
@@ -65,20 +65,26 @@ export function mergeMvnClassPaths(classPaths: string[]): string {
     .join(path.delimiter);
 }
 
-export async function getClassPathFromMvn(targetPath: string): Promise<string> {
+export async function getClassPathFromMvn(
+  targetPath: string,
+  customMavenArgs?: string[],
```

Comment:
You can have custom args with a default value of `[]`, and then remove line 74 ?

Reply:
Done